apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/BBCodingSystems.java

```diff
@@ -38,6 +20,18 @@ private BBCodingSystems() {}
 
     public static final String TAX_NUM = FISS_BASE + "/fed-tax-nb";
 
+    public static final String DCN = FISS_BASE + "/dcn";
+
+    public static final String SERV_TYP_CD = FISS_BASE + "/serv-typ-cd";
+
+    public static final String FREQ_CD = FISS_BASE + "/freq-cd";
+
+    public static final String MEDA_PROV_6 = FISS_BASE + "/meda-prov-6";
+
+    public static final String LOB_CD = FISS_BASE + "/lob-cd";
+
+    public static final String DIAG_POA_IND = FISS_BASE + "/diag-poa-ind";
+
```

Comment:
Might `FISS` and `MCS` make good enums?  Then you can dispense with the private constructor stuff and they would be more obviously related with each other rather than just collections of static final strings.

Reply:
I personally feel that enums would make it more over complicated.  Off hand, I believe it would require more code to implement, more code to utilize, and the way they are used doesn't really lend to an enum.